emcc.py

```diff
@@ -1894,6 +1896,12 @@ def include_and_export(name):
     if shared.Settings.USE_PTHREADS:
       newargs.append('-pthread')
 
+    # Any "pointers" passed to JS will now be i64's, in both modes.
+    if shared.Settings.MEMORY64:
+      if getattr(settings_key_changes, 'WASM_BIGINT', '') == '0':
+        exit_with_error('MEMORY64 is not compatible with WASM_BIGINT=0')
```

Comment:
I think you can write this as `settings_key_changes.get('WASM_BIGINT') == '0'`

Reply:
Hah, the `getattr` call was copied from one of your previous comments! And I had to add the `,''` because otherwise it would fail when the key is not there. But `get` returns `None` in that case, so I guess it can work.